src/App.js

```diff
@@ -40,31 +41,47 @@ const people = [
 const headers = ["First Name", "Last Name"];
 
 const App = () => {
-  const { selected, handleSelectOne, handleSelectAll } = useRowSelection(
-    people.length
-  );
+  const selectedValues = useRef({});;
```

Comment:
If you don't need to rerender the parent and just need these values to submit to the api then this is a rare case where you would just use a ref.